features/wasm-bulk-memory.yml

```
@@ -0,0 +1,6 @@
+name: WebAssembly bulk memory operations
+description: An extension to WebAssembly adding bulk memory operations and conditional segment initialization.
```

I wish I had any idea what this is about. I'm guessing it's something like…

```suggestion
description: Bulk memory and conditional segment initialization instructons [verb] the [nouns] for [some outcome].
```

(This is where having spec URLs, even exceptional ones, would be very helpful!)